tests/integration_tests/ingest/test_ingest.py

```diff
@@ -116,7 +122,8 @@ def test_ingest_events_when_event_type_invalid(
         "/api/v1/ingest/UNFCCC.corpus.i00000001.n0000",
         files={
             "new_data": open(
-                "tests/integration_tests/ingest/test_invalid_event_type.json", "rb"
+                "tests/integration_tests/ingest/test_bulk_data_with_invalid_event_type.json",
```

Comment:
Can we join the test path properly using os.path.join please to make sure the path parts are strung together in a way that is cross platform supported?
`import os`
```suggestion
                os.path.join("tests", "integration_tests", "ingest", "test_bulk_data_with_invalid_event_type.json"),
```